PROJECTS.md

```diff
@@ -0,0 +1,48 @@
+# OpenJS Foundation Projects
+
+The projects below are full OpenJS Foundation members. They are at the stage level indicated in the headings. Projects may move to new stages upon annual review or by request. 
+
+For information on the stages, please see the [Project Progression Document](/PROJECT_PROGRESSION.md). Links will navigate you to the project's official website.
+
+## Impact Projects
+
+* [Appium](http://appium.io/)
+* [Dojo](https://dojo.io/)
+* [jQuery](https://jquery.com/)
+* [Node.js](https://nodejs.dev/)
+* [webpack](https://webpack.js.org/)
+
+## Growth Projects
+
+* [Architect](https://arc.codes/)
+* [Intern](https://theintern.io/)
+* [MochaJS](https://mochajs.org/)
+* [Node-RED](https://nodered.org/)
+* [WebdriverIO](https://webdriver.io/)
+* [webhint](https://webhint.io/)
+
+## At Large Projects
+
+* [ESLint](https://eslint.org/)
+* [Esprima](http://esprima.org/)
+* [Express](https://expressjs.com/)
```

Comment:
Still trying to get feedback from Doug on this front.

Reply:
per conversation in meeting today, moving Express to 'TBD'